### ISSUE TITLE
UUID doesn't work, getting infi.clickhouse_orm.database.ServerError: Too large value for FixedString(16)

### ISSUE THREAD

**Issue by kpotehin** [Infinidat/infi.clickhouse_orm#102]
Hey!

I've defined the custom uuid field type as described in docs but it doesn't work. It's easy to check using existing tests, function `test_uuid_field`.
https://github.com/Infinidat/infi.clickhouse_orm/blob/84fc270e0ef04f057c981cd84cbe782eadf2d83c/tests/test_custom_fields.py

If you replace `{12345678-1234-5678-1234-567812345678}` at line 41 with `{5250a2ff-dac5-4c78-b429-d0fd079dc410}` (just a random valid uuid v4) it won't work. The error stack is below:

```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "<input>", line 38, in test_uuid_field
  File "/lib/python3.6/site-packages/infi/clickhouse_orm/database.py", line 206, in insert
    self._send(gen())
  File "/lib/python3.6/site-packages/infi/clickhouse_orm/database.py", line 324, in _send
    raise ServerError(r.text)
infi.clickhouse_orm.database.ServerError: Too large value for FixedString(16).

Row 1:
Column 0,   name: i, type: Int16,           parsed text: "0"
Column 1,   name: f, type: FixedString(16), parsed text: "RP¢ÿÚÅLx´)Ðý<0x07>Ä<0x10>"ERROR
```

Is it possible to use built-in `UUIDStringToNum` and `UUIDNumToString` functions upon inserting uuid?

I'm using Python 3.6.

Thanks!

**Comment by kpotehin** [comment]
I've fixed the issue by implementing built-in UUID ClickHouse data type.

https://github.com/Infinidat/infi.clickhouse_orm/pull/103

**Comment by ishirav** [comment]
#103 was included in v1.1.0
